CMakeLists.txt

```diff
@@ -67,8 +67,15 @@ include(cmake/SetupPackages.cmake)
 include(cmake/SetupRajaConfig.cmake)
 
 # Configure a header file with all the variables we found.
+get_directory_property( RAJA_DEFINITIONS_INITIAL DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS )
+set(RAJA_DEFINITIONS "")
+foreach(definition ${RAJA_DEFINITIONS_INITIAL})
+  set(RAJA_DEFINITIONS "${RAJA_DEFINITIONS} -D${definition}")
+endforeach()
```

Comment:
What things does this set up? The `RAJA_ENABLE` stuff, or the actual compiler flags we use?